tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRFile.java

```diff
@@ -195,6 +195,7 @@ public void testExtraPaths() {
                                     || path.startsWith("//ldml/localeDisplayNames/scripts/script")
                                     || path.startsWith(
                                             "//ldml/localeDisplayNames/territories/territory")
+                                    || path.startsWith("//ldml/localeDisplayNames/variants/variant")
```

Comment:
This is in testExtraPaths. Adding the VARIANT paths here enables the test to pass, given that `german.getExtraPaths()` now includes VARIANT paths. PlaceholderStatus appears to depend on the file Placeholders.txt, but I don't understand what it means. It seems plausible that whatever PlaceholderStatus means, isn't changed by the fact that VARIANT paths are now implemented using the ExtraPaths mechanism and they no longer have code-fallback values.

Reply:
PlaceholderStatus tells whether a value for that path can have placeholders or not. (Cf. https://cldr.unicode.org/development/updating-dtds#placeholders)
```   
public enum PlaceholderStatus {
        DISALLOWED("No placeholders allowed."), //
        REQUIRED("Specific number of placeholders allowed."), //
        LOCALE_DEPENDENT("Some placeholders may be omitted in certain locales"), //
        MULTIPLE(
                "May have multiple instances of the same placeholder, eg “{0} cats and {0} dogs”."), //
        OPTIONAL(
                "Any specific placeholder is optional (and non-numeric); there must be at least one.") //
    ;
```
It is data-driven, from Placeholders.txt. If the path doesn't match in the file (which it doesn't for variants), the value is DISALLOWED.

Reply:
Thanks! That makes sense. So the change to `testExtraPaths` seems appropriate. As that list of `path.startsWith(...)` conditions grows, the test should maybe be refactored to move that part into a subroutine.